design-system/package.json

```diff
@@ -23,7 +23,7 @@
     "build:tokens:watch": "nodemon -e yaml --quiet --watch materials/internals --exec 'yarn build:tokens'"
   },
   "dependencies": {
-    "@babel/runtime": "7.17.2",
-    "@babel/runtime-corejs3": "7.17.2"
+    "@babel/runtime": "^7.17.2",
+    "@babel/runtime-corejs3": "^7.17.2"
```

Comment:
I took the chance also to "loosen up" the version range a bit, to avoid possible version conflicts from consumers.